src/main/java/org/simplejavamail/outlookmessageparser/model/OutlookRecipient.java

```diff
@@ -17,17 +17,20 @@ public class OutlookRecipient {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(OutlookRecipient.class);
 
+	private static final String X500_ADDRESS_PATTERN = "/o=[^/]+/ou=[^/]+(?:/cn=[^/]+)*";
+
 	/**
 	 * Contains all properties that are not covered by the special properties.
 	 */
 	private final Map<Integer, Object> properties = new TreeMap<>();
 
 	private String name;
 	private String address;
+	private String x500Address;
 
 	// while parsing new properties, in some use cases the only emailaddress we get is actually encoded as name in the email
 	// but if not, we should know when to replace it with the actually encoded address, where this flag helps us
-	private boolean nameWasUsedAsAddress = false;
+	private boolean nameWasUsedAsAddress;
```

Comment:
The initialisation of nameWasUsedAsAddress=false is better to stay?

Reply:
Ahh I see. Primitive booleans already default to false; the assignment did nothing. So it's all good 👍